app/src/main/java/com/ls/ui/fragment/LocationFragment.java

```diff
@@ -110,23 +118,36 @@ private void fillMapViews(List<Location> locations) {
         uiSettings.setMyLocationButtonEnabled(false);
         uiSettings.setCompassEnabled(false);
         uiSettings.setRotateGesturesEnabled(false);
+
+        mGoogleMap.setOnMarkerClickListener(new GoogleMap.OnMarkerClickListener() {
+
+            @Override
+            public boolean onMarkerClick(Marker marker) {
```

Comment:
@shailesh351 the text is shown above, so I think no need to display window. Maybe could change colour so it shows which market is clicked.